examples/android_kts/build.gradle.kts

```diff
@@ -6,12 +6,12 @@ buildscript {
     dependencies {
         classpath("com.android.tools.build:gradle:${Versions.ANDROID_GRADLE_PLUGIN}")
         classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${Versions.KOTLIN}")
-        classpath("org.jetbrains.kotlinx:kover:${Versions.KOVER}")
+        classpath("org.jetbrains.kotlinx:kover:0.6.1")
```

Comment:
The key here is not to forget update this version once in a while, not sure how we are going to ensure it